### ISSUE TITLE
Automate collection tagging in ACT

### ISSUE THREAD

**Issue by nicolabingham** [ukwa/w3act#578]
When we build special collections in ACT we often include existing Targets as members of that collection. There is currently no automated way to do this. The process involves a curator idenfying targets, opening them in Edit mode, adding the Target to the collection and saving the record. There can several hundred Targets to be retagged in this way and given the speed of ACT this is a resource intensive and time consuming process. 
This request is to look at a way of automating this process. After talking with @anjackson we may look at this ticket as a one off scripting exercise to populate a "Football" special collection.

**Comment by anjackson** [comment]
One way to do this would be to extend the Edit Collection page so a list of Targets can be edited from there. As the list is very long, it would have to be an 'autocomplete' list rather than a full list of all items. Would that be an acceptable approach?

Would you say that, in general, it would make sense to edit Collections and add Targets to them rather than editing Targets and adding them to Collections? i.e. should Collection-editing be the main way we do this?

**Comment by nicolabingham** [comment]
@anjackson yes, it would be preferable to pull targets into a collection, rather than opening up targets individually to add them to collections. 
The autocomplete list would be an improvment on the current workflow.
